cdap-common/src/test/java/co/cask/cdap/common/conf/ArtifactConfigReaderTest.java

```diff
@@ -108,6 +108,16 @@ public void testMissingPluginFields() throws IOException, InvalidArtifactExcepti
     configReader.read(Id.Namespace.SYSTEM, configFile);
   }
 
+  @Test
+  public void testMissingPlugins() throws IOException, InvalidArtifactException {
+    // plugin JSON files written by cdap-maven-plugin does not have plugins information
```

Comment:
It's not because they're built by cdap-maven-plugin. It's because the plugins field is optional and only used in cases where the user is not building the plugin (like for jdbc drivers).

Reply:
👍